### DISCUSSION TITLE
GitHub API Issue: "updated_at" Field Not Reflecting Reaction Changes in Issues/Comments

### DISCUSSION THREAD

**Discussion by matej-kaska** [community/community#79024] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Hello GitHub Community,

I've encountered a potential issue with the GitHub API concerning the "updated_at" field in issue and comment data. During my work on a project that involves requesting data from the GitHub API, I observed that the "updated_at" timestamp does not update when reactions (like thumbs up, thumbs down, etc.) are added to an issue or a comment.

Endpoint: https://api.github.com/repos/{owner}/{repo}/issues

Specifically, when a reaction is added to an issue or a comment in a repository, I would expect the "updated_at" field associated with that issue or comment to reflect this change. However, it appears to remain unchanged.

This behavior is impacting my project as I rely on the "updated_at" field to minimize unnecessary API calls and ensure my application remains efficient and up-to-date. The current situation leads to either additional API requests (to check for reaction updates) or potential missed updates in the application.

I believe this could be considered a bug, as the addition of reactions seems like an update that should be reflected in the "updated_at" field. I am keen to hear the community's thoughts on this or learn if there is an alternative approach I should consider.

Thank you for your attention to this matter.

**Comment by ajcoll5** [comment]
Agreed that this should be a bug - the inconsistency in when this date field is updated is a problem for my script monitoring commits and new issues within an org.
